pkgs/tools/system/ipmitool/default.nix

```diff
@@ -22,6 +22,11 @@ stdenv.mkDerivation {
       url = "https://github.com/ipmitool/ipmitool/commit/5db314f694f75c575cd7c9ffe9ee57aaf3a88866.patch";
       sha256 = "01niwrgajhrdhl441gzmw6v1r1yc3i8kn98db4b6smfn5fwdp1pa";
     })
+    # CVE-2020-5208 fix
```

Comment:
If you instead set the `name = ` argument of  `fetchpatch` to something like `CVE-2020-5208.patch`, (some) automated tools are able to pick this up and mark this issue as fixed...